python/feathub/feature_tables/sinks/redis_sink.py

```diff
@@ -41,7 +42,16 @@ def __init__(
         :param namespace: The namespace to persist features in Redis. Feature tables
                           sinking to Redis sinks with different namespaces can save
                           records with the same key into Redis without overwriting
-                          each other.
+                          each other. Only valid when key_expr is not set.
+        :param key_expr: A string that represents a FeatHub expression which evaluates
+                         to a string value, which would be used as the key to a feature
+                         saved in Redis. Apart from the field names, UDFs and other
+                         grammars supported by Feathub expression, users may also use
+                         the keyword, "__FEATURE_NAME__", in this expression to
```

Comment:
It might be useful to also provide `__NAMESPCE__`, so that users do not need to specify different `key_expr` for Redis Sinks with different namespaces.